parser/src/cfg/defcfg.rs

```diff
@@ -201,11 +201,11 @@ impl Default for CfgOptions {
             chords_v2_min_idle: 5,
             #[cfg(any(
                 all(target_os = "windows", feature = "interception_driver"),
-                target_os = "linux",
+                any(target_os = "linux", target_os = "android"),
                 target_os = "unknown"
             ))]
             mouse_movement_key: None,
-            #[cfg(any(target_os = "linux", target_os = "unknown"))]
+            #[cfg(any(any(target_os = "linux", target_os = "android"), target_os = "unknown"))]
```

Comment:
No need for nested `any` here, can just add `target_os = "android"` to the comma separated list.